py3status/modules/spotify.py

```diff
@@ -150,8 +153,10 @@ def _get_text(self):
                 )
                 if playback_status.strip() == "Playing":
                     color = self.py3.COLOR_PLAYING or self.py3.COLOR_GOOD
+                    paused = False
```

Comment:
Thoughts on passing string `playback_status` instead of boolean `paused`? (i.e. `\?if=playback=Paused`, `\?if=playback=Playing`, etc). This ought allow users to print `Playing`, `Paused`, `Stopped`, etc too (untested).

Reply:
This definitely sounds more intuitive than my boolean.
I just checked other modules for media players and it looks like most of them have `state` placeholder. Some of them also allow user to set `state_play`, `state_pause` etc. strings but I don't think this is necessary given you can always just use `\?if=state`.

I would be happy to change that.

Reply:
Nice plan @lasers 

Just wondering if it matches our goal in the long run but don't have a strong opinion on this.

Let's say that this is the way to do it right now, so please adapt for this @bcyran

Reply:
>Just wondering if it matches our goal in the long run but don't have a strong opinion on this.

It's what I'd do... Boolean placeholders just mean more placeholders. We can use formatter conditionals on a single placeholders instead of creating more placeholders. Sure, it's only three more placeholders for `playback`, but it could be more than three for any other placeholders.

Lot of modules using `state_` was done long before the formatter conditionals too.